civictechprojects/static/css/partials/_AboutProjectEventDisplay.scss

```diff
@@ -18,8 +18,16 @@
     button,
     a {
       width: 100%;
+      margin: 1rem 0;
```

Comment:
note to self; remove this and use `gap: 1rem;` or whatever value on parent flex container (Profile-top-interactions)